### ISSUE TITLE
Failing when using git tags only

### ISSUE THREAD

**Issue by stuartf7** [TriPSs/conventional-changelog-action#46]
I'm using this action with a golang service that I've built, so I'm trying to configure the action to use tags rather than a file, I've setup the following step in my workflow.
```
      - name: Conventional Changelog Action
        id: changelog
        uses: TriPSs/conventional-changelog-action@v3
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          output-file: 'false'
          skip-commit: 'true'
```

It's a clean repository, I've tried running without any existing tags and I've also tried pushing a v0.0.0 tag to a previous commit too, but it always seems to result in the following error:

```
Using "v" as tag prefix
Using "false" as output file
Skipping empty releases is "enabled"
Skipping the update of the version file is "disabled"
Pull to make sure we have the full git history
/usr/bin/git remote set-url origin ***github.com/******/service-registration.git
/usr/bin/git pull --unshallow --ff-only
From https://github.com/******/service-registration
 * [new branch]      build/test           -> origin/build/test
 * [new branch]      feature/initial      -> origin/feature/initial
 * [new branch]      feature/registration -> origin/feature/registration
Already up to date.
Recommended release type: minor
Because: There are 0 BREAKING CHANGES and 1 features
Using GIT to determine the new version
/home/runner/work/_actions/TriPSs/conventional-changelog-action/v3/src/version/git.js:16
        const currentVersion = tags.shift().replace(tagPrefix, '')
                                           ^

TypeError: Cannot read property 'replace' of undefined
    at /home/runner/work/_actions/TriPSs/conventional-changelog-action/v3/src/version/git.js:16:44
    at /home/runner/work/_actions/TriPSs/conventional-changelog-action/v3/node_modules/git-semver-tags/index.js:59:5
    at ChildProcess.exithandler (child_process.js:286:7)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```

**Comment by TriPSs** [comment]
Hi @stuartf7,

Thanks for the issue, I will try to take a look today!

**Comment by TriPSs** [comment]
This is released in [v3.3.0](https://github.com/TriPSs/conventional-changelog-action/releases/tag/v3.3.0)

**Comment by TriPSs** [comment]
Thank you @Jayrgo for the contribution!
